test/orfs/mock-array/mock-array.bzl

```diff
@@ -219,6 +219,7 @@ MACROS = [
 ]
 
 POWER_TESTS = [
+    "openroad",
```

Comment:
@jhkim-pii This PR has new tests running power tests with openroad binary in addition to sta binary